scripts/npcs/village/durmark.lua

```diff
@@ -113,7 +113,7 @@ local function durmark_talk(npc, ch)
     end
 
     local function about_reward()
-        say("Ah! thank's a lot!")
+        say("Ah! thanks a lot!")
```

Comment:
Should be "Thanks", with uppercase T. :-)

Reply:
Fixed it!